pactl/pactl.go

```diff
@@ -7,6 +7,15 @@ import (
 	"github.com/l3lackShark/audio-share-discord-linux/shell"
 )
 
+//Checks for pulseaudio
+func CheckForPulseaudio() bool {
+	raw := string(shell.Cmd("pactl --version", true))
```

Comment:
are you sure that this is correct way of checking this? Wouldn't `pulseaudio --version` or something like `ps aux | grep pulseaudio` be better to make sure that PA is currently running?

Reply:
Actually `pgrep pulseaudio` would be perfect here I think.

Reply:
Yeah that's probably the better way to do about it